components/shared/atoms/Input/Input.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+export const Input = styled.input`
+  width: 100%;
+  padding: 1rem;
+  font-size: 1rem;
+  border-radius: 0.3rem;
+  border: 1px solid rgb(179, 179, 179);
```

Comment:
Could you please use theme variables for colors?

Reply:
Okay. I copied it from another button. I'll replace all places with theme colors